apps/dapp-console-api/src/constants/envVars.ts

```diff
@@ -206,5 +221,17 @@ export const envVars = envVarSchema.parse(
         CB_VERIFICATION_EAS_API_URL: process.env.CB_VERIFICATION_EAS_API_URL,
         CB_VERIFICATION_SCHEMA_ID: process.env.CB_VERIFICATION_SCHEMA_ID,
         CB_VERIFICATION_ATTESTER: process.env.CB_VERIFICATION_ATTESTER,
+        FAUCET_V1_CONTRACT_ADDRESS:
+          process.env.FAUCET_V1_CONTRACT_ADDRESS ??
+          '0x6f324a7306c430489941990A25bA7268a69fd63e',
+        FAUCET_V1_AUTH_ADMIN_WALLET_PRIVATE_KEY:
+          process.env.FAUCET_V1_AUTH_ADMIN_WALLET_PRIVATE_KEY ??
+          // Account 0 on Anvil
+          '0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80',
```

Comment:
nit: I would just do:
```
FAUCET_V1_AUTH_ADMIN_WALLET_PRIVATE_KEY:
          process.env.FAUCET_V1_AUTH_ADMIN_WALLET_PRIVATE_KEY
```
and not default to the anvil account. I think we originally did this in gateway as a stop gap while we waited for the secret to be set up.